about.html

```diff
@@ -0,0 +1,150 @@
+                    </h2>
+                    <p class="summits-text">
+                        Take a look at the last two Digital Marketing
+                        Summit which took place in London and in Texas
+                    </p>
+                    <ul class="summits-list">
+                        <li class="summits-list-item">
+                            <img src="./images/digimarcon-america-new-conference-2.jpg" alt="A picture showing the past Digital Marketing Summit summit in London">
+                            <div class="overlay">
+                            </div>
+                            <div class="summits-list-item__details" >
+                                <h6>2020</h6>
+                                <p>Digital Marketing Summits 2020 in London</p>
+                            </div>
+                        </li>
+                        <li class="summits-list-item">
+                            <img src="./images/shutterstock_1117902230.jpg" alt="A picture showing the past Digital Marketing summit in Texas">
+                            <div class="overlay">
+                            </div>
+                            <div class="summits-list-item__details" >
+                                <h6>2019</h6>
+                                <p>Digital Marketing Summits 2019 in Texas</p>
+                            </div>
+                        </li>
+                    </ul>
+                </div>  
+            </div>
+        </section>
+        <section>
+            <div class="container bg-t">
+                <div class="partner">
+                    <h2 class="partner-heading">Partner</h2>
+                    <ul class="partners-list">
+                        <li class="partner-list-item">
+                            <i class="fab fa-facebook"></i>
+                        </li>
+                        <li class="partner-list-item">
+                            <i class="fab fa-twitter"></i>
+                        </li>
+                        <li class="partner-list-item">
+                            <i class="fab fa-amazon"></i>
+                        </li>
+                        <li class="partner-list-item">
+                            <i class="fab fa-airbnb"></i> 
+                        </li>
+                        <li class="partner-list-item">
+                            <i class="fab fa-google"></i> 
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </section>
+        
+    </main>
+    <footer>
+        <div class="container">
+            <div class="footer">
+                <div class="logo">
+                    <a  href="index.html">
+                       <h2>CGDMC.2021</h2>  
+                    </a>
+                </div>    
+                <p class="rights">
+                    @ Cairo International Convention Center.
+                </p>
+            </div>
```

Comment:
- [ ] On the footer in the about page, kindly ensure you use the exact background color as seen in the original.

| Original | Yours |
| ----- | ------ |
| <img src='https://user-images.githubusercontent.com/33323077/128996362-af6e4a21-3257-4d0e-bf71-de7a8806c828.png' height='250px'> | <img src='https://user-images.githubusercontent.com/33323077/128996496-277e790e-6fbe-4811-ac99-abdcf54b82df.png' height='250px'> |